linetools/scripts/lt_continuumfit.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+"""
+Enable continuum fitting of an input spectrum
+"""
+import pdb
+import sys
+
+
+# Script to run continuum fitting in linetools
+def main(*args, **kwargs):
+    """ Runs the continuum fitter
+    """
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Parse for Continuum Fitting')
```

Comment:
The description string is what shows up when you call 'lt_continuum -h'. So it should say what the script does, i.e. "Fit a continuum to a spectrum".